# Patch com_google_protobuf to define //:descriptor_proto_srcs. Since
# TensorFlow already patches protobuf and tf_http_archive doesn't support
# multiple patch files, we instead augment their patch.
--- third_party/protobuf/protobuf.patch
+++ third_party/protobuf/protobuf.patch
@@ -19,2 +19,12 @@
          "//build_defs:config_msvc": [],
+@@ -172,3 +174,9 @@
+ )
+
++filegroup(
++    name = "descriptor_proto_srcs",
++    srcs = ["src/google/protobuf/descriptor.proto"],
++    visibility = ["//visibility:public"],
++)
++
+ filegroup(
 diff --git a/python/google/protobuf/pyext/descriptor.cc b/python/google/protobuf/pyext/descriptor.cc
